tests/API_smoke_test.postman_collection.json

```diff
@@ -2109,40 +2113,54 @@
 							"response": []
 						},
 						{
-							"name": "Logout of tidepool",
+							"name": "Delete User",
 							"event": [
 								{
 									"listen": "test",
 									"script": {
-										"id": "e7fae2fe-0a46-4c31-aa87-6d6fd8f915a6",
+										"id": "4a976bef-4188-4fc2-88f7-dc79f18a48e7",
 										"exec": [
-											"const assert = eval(globals.loadAsserts);",
-											"assert.common(200, 300, false, null, \"\");"
+											"if (pm.environment.get(\"api\") == \"https://api.tidepool.org\") {\r",
+											"  pm.test(\"Delete user successful, timeout is expected\", function () {\r",
```

Comment:
so we actually expect a failure on PRD ?

Reply:
yep. It essentially stems from the root of how the delete user api is currently functioning. Its able to return back successfully on qa1 and qa2 because the database is smaller. on prd, well, it times out. This is the ticket to turn it into a background process rather than trying to delete it at that moment and return a success message back to the caller, which is essentially what we do now. https://tidepool.atlassian.net/browse/BACK-1135